pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/thread/ThreadRuntime.java

```diff
@@ -149,7 +149,7 @@ private static ClassLoader loadJars(String jarFile,
             } catch (FileNotFoundException e) {
                 // this is usually like
                 // java.io.FileNotFoundException: /tmp/pulsar-nar/xxx.jar-unpacked/xxxxx/META-INF/MANIFEST.MF'
-                log.error("The file {} does not look like a .nar file", jarFile, e.toString());
+                log.error("The file {} does not look like a .nar file {}", jarFile, e.toString());
```

Comment:
@eolivelli - was your intention in https://github.com/apache/pulsar/pull/12667 to avoid logging the error's stack trace? If not, we could also drop the `.toString()` to make this log line work.

```suggestion
                log.error("The file {} does not look like a .nar file", jarFile, e);
```